be1-go/test/json_test_strings.txt

```diff
@@ -1,23 +0,0 @@
-String for publishing LAO creation:

```

Comment:
why remove this ? 
The hashes are probably wrong now, but it's worth updating and could be useful as a starting point.